### ISSUE TITLE
Error while building project

### ISSUE THREAD

**Issue by joaotnlima** [sujaykundu777/devlopr-jekyll#16]
Hello, nice project - i was trying to check it, but while building it this error ocurred:

```
Fetching algolia_html_extractor 2.6.2
Errno::EACCES: Permission denied @ rb_file_s_rename -
(/home/jekyll/.gem/ruby/2.7.0/cache/algolia_html_extractor-2.6.2.gem,
/usr/gem/cache/algolia_html_extractor-2.6.2.gem)
An error occurred while installing algolia_html_extractor (2.6.2), and Bundler
cannot continue.
Make sure that `gem install algolia_html_extractor -v '2.6.2' --source
'https://rubygems.org/'` succeeds before bundling.

In Gemfile:
  jekyll-algolia was resolved to 1.6.0, which depends on
    algolia_html_extractor

```

Can you help?

**Comment by sujaykundu777** [comment]
@joaotnlima Can you try install the gems locally by `bundle update && bundle install` , this will install all the necessary dependencies. 

Then you can run `bundle exec jekyll serve` to check the site build locally.

**Comment by joaotnlima** [comment]
```
Installing json 2.3.0 with native extensions
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

current directory:
/tmp/bundler20200427-19681-1toxzccjson-2.3.0/gems/json-2.3.0/ext/json/ext/generator
/usr/bin/ruby2.5 -r ./siteconf20200427-19681-1xfjr9c.rb extconf.rb
mkmf.rb can't find header files for ruby at /usr/lib/ruby/include/ruby.h

extconf failed, exit code 1

Gem files will remain installed in
/tmp/bundler20200427-19681-1toxzccjson-2.3.0/gems/json-2.3.0 for inspection.
Results logged to
/tmp/bundler20200427-19681-1toxzccjson-2.3.0/extensions/x86_64-linux/2.5.0/json-2.3.0/gem_make.out

An error occurred while installing json (2.3.0), and Bundler cannot
continue.
Make sure that `gem install json -v '2.3.0'` succeeds before bundling.

In Gemfile:
  jekyll-algolia was resolved to 1.6.0, which depends on
    algolia_html_extractor was resolved to 2.6.2, which depends on
      json
```

**Comment by sujaykundu777** [comment]
@joaotnlima  can you share the versions of the following that's been installed in your system- 

OS name and version
ruby -v
bundler -v 
bundle exec jekyll -v
